model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -255,6 +259,15 @@ def count_tokens(input: str, model_name: str, tokenizer_repository: TokenizerRep
     return len(tokenizer.encode(input))
 
 
+async def _get_latest_tag(inference_framework: LLMInferenceFramework) -> str:
+    config_map = await read_config_map(LATEST_INFERENCE_FRAMEWORK_CONFIG_MAP_NAME)
+    if inference_framework not in config_map:
+        raise LatestImageTagNotFoundException(
+            f"Could not find latest tag for inference framework {inference_framework}."
+        )
+    return config_map[inference_framework]
```

Comment:
let's also check whether the image actually exists

Reply:
this happens later in the creation process

Reply:
you meant within `create_llm_model_bundle_use_case.execute`?

Reply:
we first map "latest" to the value from the config map. then in `create_llm_model_bundle_use_case.execute` that tag is checked